source/documentNavigation/sentenceHelper.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: UTF-8 -*-
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2022 NV Access Limited, Rob Meredith
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+import re
+from typing import Optional
+
+WESTERN_TERMINATORS_PATTERN: re.Pattern = re.compile("[.?!](?: |\r|\n)+")
+WESTERN_TERMINATORS_SINGLE_PATTERN: re.Pattern = re.compile("[.?!]")
+FULL_WIDTH_TERMINATORS_PATTERN: re.Pattern = re.compile("[。！？](?:\r|\n)*")
+
+
+def _findEndOfSentence(sentence: str, offset: int) -> Optional[int]:
```

Comment:
Actually, we try to find the next end of sentence pattern in a larger text. What about renaming as follows:

```suggestion
def _findNextEndOfSentence(text: str, offset: int) -> Optional[int]:
```